bitnami/mongodb/values.yaml

```diff
@@ -123,6 +123,9 @@ schedulerName: ""
 ## @param architecture MongoDB&reg; architecture (`standalone` or `replicaset`)
 ##
 architecture: standalone
+
+bypassUpgradeValidation: false
+
```

Comment:
Can't we implement the logic to detect when to bypass the validations instead? I mean, we can simply detect that Vault is being used to manage the credentials and skip the validations.

Reply:
This is for the upgrade use case. When we try to upgrade it from the existing deployment where the credentials are already saved as Mongo Env Vars.

Reply:
I don't find the parameter useful yet. I mean, regardless I'm upgrading from an existing deployment with credentials as env. vars or already using Vault, why can't I simply assume that validations should be skipped when Vault is enabled?